site/src/utils/url.js

```diff
@@ -6,9 +6,10 @@ export function parseTwitterHandle(url) {
   let handle
 
   try {
-    handle = url.match(TwitterRegexp)[1]
+    const [match] = url.match(TwitterRegexp)[1]
```

Comment:
Is this functionally equivalent? It looks like we're double-destructuring now

Reply:
I made it clearer now :)